.bcr/presubmit.yml

```diff
@@ -0,0 +1,12 @@
+bcr_test_module:
+  module_path: "e2e/bzlmod"
+  matrix:
+    platform: ["macos"]
+  tasks:
+    build_flags:
+      - "--config=ios"
+    run_tests:
+      name: "Run test module"
+      platform: ${{ platform }}
+      build_targets:
+        - "//..."
```

Comment:
Our release archive doesnt contain the `tests` directory so we just care about building here. This is similar to other repos that maintain minimal release archives. 

Relying on rules_ios CI is good enough IMO and ensuring the release stuff builds is probably good enough as a starting point for this check.